db/migrate/20181211094456_add_indexes_for_send_emails.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexesForSendEmails < ActiveRecord::Migration[5.1]
+  def change
+    add_index :listings, [:community_id, :author_id, :deleted], name: 'person_community_exist'
+    add_index :community_memberships, [:community_id, :person_id, :status], name: 'person_community_status'
```

Comment:
As far as I could test, this index didn't contribute much. Did you see some query where this helps @aivils? Also index name could be better as `community_person_status` to reflect correct order.